plugins/talk-plugin-local-auth/translations.yml

```diff
@@ -1,4 +1,12 @@
 en:
+  email:
+    email_change_original:
+      subject: Email change
+      body: Your email address has been changed from {0} to {1}. If you did not initiate this change, please contact {2}. # TODO: update translation
```

Comment:
What needs to be updated?

Reply:
Previously, all the translations lived in two files, which didn't work for the client code as that code didn't live in the right place, this fixes that :)

Reply:
I think that content wasn't vetted by product/design so I wanted to flag it to get reviewed :)